src/Artifacts.php

```diff
@@ -98,7 +100,12 @@ public function download(Tags $tags, array $configuration): array
         }
 
         if (!empty($configuration['artifacts']['shared']['enabled'])) {
-            return $this->downloadShared($tags->setIsShared(true), $isArchive);
+            $filter = $configuration['artifacts']['custom']['filter'];
+            return $this->downloadShared(
+                $tags->setIsShared(true),
+                $filter['limit'] ?? null,
```

Comment:
přidal jsem do shared artefaktů taky ten limit, nemyslím si, že by to něčemu vadilo a je to o chloupek konzistentnější

Reply:
Dobre